wiki/Configure-lichess-bot.md

```diff
@@ -1,6 +1,19 @@
 # Configuring lichess-bot
 There are many possible options within `config.yml` for configuring lichess-bot.
 
+## Note on daily game limits
+
+Lichess allows a bot to play 100 games against other bots in a single day (games against humans are unlimited). Several settings below can influence how quickly this game allowance is used up. These include the following:
+- `challenge:`
+  - `concurrency`: Playing multiple simultaneous games will use up games faster.
+  - `accept bot` and `only_bot`: Only bot games use up the allotted games.
```

Comment:
`accept bot` needs an underscore. `Only bot games use up the allotted games.` seems a bit confusing to me. The best change I found would be to remove the `Only` as it can be confused with `only_bots`.